### ISSUE TITLE
Notes with long text become sluggish

### ISSUE THREAD

**Issue by raaphorst** [ttscoff/nv#481]
When you have large amounts of text NVALT starts to respond sluggishly. I have one note with 6800 words and I see the letters appear not instantly on the screen but sluggishly.

**Comment by ttscoff** [comment]
If you haven't tried it already, disable live word count and see if the issue resolves.

**Comment by raaphorst** [comment]
thanks. that solves it!

**Comment by chris-zettelkasten** [comment]
Unfortunately I have the same issue without using word count.

**Comment by bunk3m** [comment]
Hi. I have noticed a similar issue.  

I have about 450 notes taking up 1.3MB.  When I scroll down the list of notes (in the view with the note list in the top and a window showing the contents of the note below), the contents of the next note isn't shown for upwards of 30+ seconds.  This has only started recently.  It is quite frustrating.

I don't know if this will have any relevance to the issue at hand, but I sync my local notes folder (Library:Application Support:Notional Data) to a server so I can have access to my notes on an iMac and MBP.  I use Unison to sync local content to the server and back.

I am using nvAlt v2.2.8 (128) on Mac OSX 10.12.6 (iMac) and 10.11.6 (MBP)

**Comment by ttscoff** [comment]
Do you have the preview open when switching notes?

**Comment by bunk3m** [comment]
@ttscoff Nope.  Preview is off/closed.  (Assume you mean the window that opens when you press ^+CMD+P?)

I do have the note window open. ie. the one below the note list. 
Picture
<img width="407" alt="screen shot 2018-02-25 at 19 11 35" src="https://user-images.githubusercontent.com/3494567/36648314-cefd0e58-1a5f-11e8-96ef-2fbf24b420c6.png">
